agile-blanche/src/app/components/BlobButton/BlobButton.tsx

```diff
@@ -0,0 +1,38 @@
+'use client';
+
+import './BlobButton.css';
+
+interface BlobButtonProps {
+    text?: string;
+    onClick?: () => void;
+    isBlack?: boolean;
+    isWhite?: boolean;
+    isHome?: boolean;
+    isNav?: boolean;
+}
+
+export const BlobButton = ({isNav = false, isHome = false, isWhite = false, isBlack = false, text = 'Blob Button', onClick }: BlobButtonProps) => {
```

Comment:
Because we're changing `text` to be required, you can remove the initial value too?

```suggestion
export const BlobButton = ({isNav = false, isHome = false, isWhite = false, isBlack = false, text, onClick }: BlobButtonProps) => {
```

Reply:
It is now a required prop